temporalcli/commands.workflow_trace.go

```diff
@@ -0,0 +1,67 @@
+package temporalcli
+
+import (
+	"fmt"
+	"github.com/fatih/color"
+	"github.com/temporalio/cli/temporalcli/internal/printer"
+	"github.com/temporalio/cli/temporalcli/internal/trace"
+	"go.temporal.io/sdk/client"
+)
+
+func (c *TemporalWorkflowTraceCommand) run(cctx *CommandContext, _ []string) error {
```

Comment:
```suggestion
func (c *TemporalWorkflowTraceCommand) run(cctx *CommandContext, _ []string) error {
	if cctx.JSONOutput {
		return fmt.Errorf("JSON output not supported for trace command")
	}
```

Let's disable JSON for this command for now which also means you can remove JSON tags from your struct below